.bash_nav

```diff
@@ -1,9 +1,19 @@
 # alias to run sublime text from anywhere
 alias subl='"/mnt/c/Program Files/Sublime Text 3/sublime_text.exe"'
 
-# wanna go to desktop
-alias desk='cd /mnt/c/Users/gsera/Desktop'
+if [ -d '/mnt/c/Windows' ]; then
 
-# source bash
+    # alias cmd for calling the windows command line from bash
+    alias cmd='/mnt/c/Windows/System32/cmd.exe /C'
+
+    # wanna go to desktop
+    alias desk='cd /mnt/c/Users/$(cmd "echo | set /p=%USERNAME%")/Desktop'
```

Comment:
This alias opens a subshell and puts the output of the subshell into the string representing the path to your desktop. The subshell uses the `cmd` command to go into windows world, grab the user's username, and echo it to the output of the subshell.